tests/test_logging.py

```diff
@@ -159,11 +159,35 @@ def on_session_start():
         assert lines[0] in f.read().splitlines()
 
 
+@pytest.mark.parametrize('level', ['info', 'error'])
+@pytest.mark.parametrize('use_slash_logger', [True, False])
+def test_filtering_slash_logs(files_dir, suite, suite_test, config_override, level, use_slash_logger):
+    config_override('log.filter_log_level', logbook.WARNING)
+    assert slash.config.root.log.truncate_console_lines
 
+    long_string = 'a' * 1000
+    if use_slash_logger:
+        suite_test.append_line('slash.logger.{level}({msg!r})'.format(msg=long_string, level=level))
```

Comment:
`slash.logger` is exactly an example of what *not* to filter, since it's advertised as a mechanism for the use of the test itself (See http://slash.readthedocs.io/en/master/tour.html#logging)